docs/spec/api.md

```diff
@@ -1258,10 +1258,10 @@ The following parameters should be specified on the request:
 
 
 
-###### On Success: Accepted
+###### On Success: Created
 
 ```
-202 Accepted
+201 Created
```

Comment:
I don't think it does - I believe the lines that changed here are entirely generated based on descriptors.go.